### PR TITLE
fix(gameList): restore Last Updated column

### PR DESCRIPTION
**Pull request by wescopeland**
Though misleading, this seems to have been used by a lot of people.

### CODE REVIEW

File: public/gameList.php [hunk 1]
```diff
@@ -54,6 +54,7 @@ function ListGames(
         echo "<th class='text-right'><a href='/gameList.php?s=$sort2$queryParams'>Achievements</a></th>";
         echo "<th class='text-right'><a href='/gameList.php?s=$sort3$queryParams'>Points</a></th>";
         echo "<th class='text-right'><a href='/gameList.php?s=$sort7$queryParams'>Retro Ratio</a></th>";
+        echo "<th style='white-space: nowrap'><a href='/gameList.php?s=$sort6$queryParams'>Last Updated</a></th>";
```
Comment (Jamiras) [inline public/gameList.php:57]: This should be right aligned.
image

Comment (wescopeland) [reply public/gameList.php]: Thanks, the game list I was testing against had long enough game names to push the table to the point that I missed this. Resolved in https://github.com/RetroAchievements/RAWeb/pull/1841/commits/6d99280826b7804a024918739024ca28899f7e71.

File: public/gameList.php [hunk 2]
```diff
@@ -64,6 +65,7 @@ function ListGames(
         echo "<th class='text-right'>Achievements</th>";
         echo "<th class='text-right'>Points</th>";
         echo "<th class='text-right'>Retro Ratio</th>";
+        echo "<th style='white-space: nowrap'>Last Updated</th>";
```
Comment (Jamiras) [inline public/gameList.php:68]: This should be right aligned too. - Developer Sets page
image

Comment (wescopeland) [reply public/gameList.php]: Thanks, resolved in https://github.com/RetroAchievements/RAWeb/pull/1841/commits/94d47dc42d0352a238a68f2ca3f1ca58ffb97bbc.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**